pkgs/development/tools/build-managers/dub/default.nix

```diff
@@ -1,11 +1,11 @@
-{stdenv, fetchurl, curl, dmd, gcc, unzip}:
+{stdenv, fetchurl, curl, dmd, gcc, unzip, git}:
```

Comment:
dub build calls git although it does not fail without it:
https://github.com/D-Programming-Language/dub/blob/0.9.23/build.sh#L33

I think dmd build also calls git but I did not address that.